app/mailers/application_mailer.rb

```diff
@@ -3,7 +3,7 @@
 class ApplicationMailer < ActionMailer::Base
   # <!--===================-->
   # <!--to set the default mailer address-->
-  default from: 'eventnxtapp@gmail.com' # same email can be found at "/config/environments/development.rb"
+  default from: 'suryacherukuri999@gmail.com' # same email can be found at "/config/environments/development.rb"
```

Comment:
Not sure about this. I think it's better we send email from the event orignaizer email. That is, get the email from the user model, but I don't know if it is possible or not.